webtasks/cron_backend_mongodb.js

```diff
@@ -319,6 +330,71 @@ router.put('/:container/:name',
             .then(res.json.bind(res), next);
 });
 
+router.put('/:container/:name/state',
+    ensure('headers', ['host']),
+    ensure('params', ['container', 'name']),
+    function (req, res, next) {
+        var validStates = ['active', 'inactive'];
+        var data = req.webtaskContext.data;
+        var jobs = req.mongo.collection(data.JOB_COLLECTION);
+        var cluster_host = data.CLUSTER_HOST;
+        var state = req.body.state || req.query.state;
+        
+        if (validStates.indexOf(state) < 0) {
+            return next(Boom.badRequest('A job\'s state can only be set to one '
+                + 'of `' + validStates.join('`, `') + '`.'));
+        }
```

Comment:
Would it make sense to factor out state validation to a function since it is used in 2 places?